### DISCUSSION TITLE
Air unit O3, speedeybee F405 WING, INAV 7.1.2: Crosshair and horizon sidebar in OSD not in middle of google2 screen

### DISCUSSION THREAD

**Discussion by samsab1** [iNavFlight/inav-configurator#2172] [General]
hello
MSP displayport on UART5, BF43COMP on OSD; impossible to get the cross air and horizon in the middle of the screen.
is there something to do?
best regards

**Comment by mmosca** [comment]
If your goggles are setup for hd, you should BFHDCOMP instead of BF43COMP.

**Comment by samsab1** [comment]
i tried that too. same result, not centered on google2

**Comment by samsab1** [comment]
The others paramesters are diplayed where it should be, except cross hair and horizon sidebar........

**Comment by samsab1** [comment]
why do you close this thread?; it 's not solved from my point of view.

**Comment by mmosca** [comment]
Issues are for bugs. This is a configuration issue.

BF43COMP requires DJI goggles to be setup for non-hd osd.
If the goggles are setup for HD osd, you need to use the HD option instead.

**Comment by samsab1** [comment]
thanks mmosca.
on my config: O3 + googles2, i have only **one** option canvas: **HD**
so HD is applied on my google, if i understand well what you are talking about.
by the way, i have to say that on my quad with an other 03 (speedybee f405 v4, INAV 7.1.2), there is absolutly no pb with osd....that's why i  am really confused

**Comment by mmosca** [comment]
The option in both INAV and your DJI goggles need to agree. They both need to be set to HD.

**Reply by samsab1** [reply]
so can you confirm the config above?

inav 7.1.2
uart5: MSP DISPLAY
OSD: BFHD

Google2 canvas: HD

is that coherent?

**Comment by samsab1** [comment]
don't ask why, but now it works as it should.....
thanks again for your help
i was struggeling since 48 hours, but now, it works, so strange.
